app.ts

```diff
@@ -0,0 +1,106 @@
+﻿/// <reference path="jquery.d.ts" />
+
+/**
+ * Michael Leibbrandt 
+ * Typescript Onboard Project
+ */
+
+namespace onboard {
+
+	// Variable declarations
+
+	let totRec = getMax(); // Total amount of records
+	let totHeadings;
+	let fromRecord = 0;
+	let toRecord = 15;
+	let count = 0;
+	let increase = 15;
+
+	$(document).ready(function () {
+
+		// Creation of table as well as insertion of data
+
+		function populate(to, from) {
```

Comment:
Define types for the input variables, and also define a return type for the function.